doc-content/enterprise-only/project-data/multiple-branches-project-con.adoc

```diff
@@ -0,0 +1,7 @@
+[id='multiple-branches-con']
+
+= Multiple branches in {CENTRAL}
+
+Multiple branch support in {CENTRAL} provides the ability to create a new branch based on an existing one, including all of its assets. All new, imported, and sample projects open in the default *master* branch. You can create as many branches as you need and can work on multiple branches interchangeably without impacting the original project on the *master* branch.
+
+This new functionality adds support for persisting branches, which means that {CENTRAL} remembers the last branch used and will open in that branch when you log back in.
```

Comment:
While the description of "new functionality" can be correct for this version, it is a maintenance trap. What if you forget to change this formula and it remains "new" five versions later? I would suggest reformulating to avoid referring to "new functiionality" as this is not the Release Notes. Alternatively mention the version number, hardcoded (this functionality, new in version 7.1...)

Reply:
Corrected. Thanks!